server/re/include/irods_re_plugin.hpp

```diff
@@ -307,8 +307,8 @@ namespace irods {
 
         template<typename OP, typename... As >
         error call(
-            std::string _instance_name,
-            std::string _operation_name,
+            const std::string& _instance_name,
+            const std::string& _operation_name,
```

Comment:
I don't *think* this will be an issue so there is probably not an action item here, but we may want to make sure the signature change does not break the existing rule engine plugins (or, if it does, that they are updated to reflect it). I am recalling an instance where the audit REP was surprisingly broken due to an undetected change in a framework-related signature: https://github.com/irods/irods_rule_engine_plugin_audit_amqp/commit/264fbfed26a64b7775d8d3f731df92533d37a084 However, I don't see `call` being invoked anywhere in there, so it may be fine.

@korydraughn do you know if REPs might be impacted by this?

Reply:
Actually, after stepping away for 30 seconds, I don't think this would affect anything as we are *adding* a `const` qualifier. Please ignore :)